OpenRA.Mods.Common/Traits/Buildings/Building.cs

```diff
@@ -186,7 +186,7 @@ public IEnumerable<WPos> TargetablePositions(Actor self)
 		public void Created(Actor self)
 		{
 			if (SkipMakeAnimation || !self.Info.HasTraitInfo<WithMakeAnimationInfo>())
-				NotifyBuildingComplete(self);
+				self.World.AddFrameEndTask(w => NotifyBuildingComplete(self));
```

Comment:
This looks rather dangerous (i.e. could easily introduce regressions).
Wouldn't it be easier to add a simple `turret != null` check in `BuildComplete` (re https://github.com/OpenRA/OpenRA/pull/11600#discussion_r74230299)?
(And possibly set `DesiredFacing` to `null` in `Created`?)